### PR TITLE
Get correct culture on identity pages to load correct container

### PR DESCRIPTION
**Pull request by mbullin**
Do not merge until Adam gives his ok!

### CODE REVIEW

File: src/cmp.js [hunk 1]
```diff
@@ -166,7 +166,7 @@ module.exports.loadCmpAsync = once(() => {
 });
 
 module.exports.isCmpEnabled = () => {
-    return !window.__tcfapi && window.cmpEnabled;
+    return !window.__tcfapi && window.cmpEnabled && window.hostname !== "accounts.autoscout24.com";
```
Comment (ctoutountzidis) [inline src/cmp.js:169]: Did you check that? :D 
window.location.hostname

Comment (mbullin) [reply src/cmp.js:169]: argh you're right ...

File: src/cmp.js [hunk 2]
```diff
@@ -166,7 +166,7 @@ module.exports.loadCmpAsync = once(() => {
 });
 
 module.exports.isCmpEnabled = () => {
-    return !window.__tcfapi && window.cmpEnabled;
+    return !window.__tcfapi && window.cmpEnabled && window.location.hostname !== "accounts.autoscout24.com";
```
[the remark this answers is not in the document]
Comment (bdadam) [reply src/cmp.js:169]: We only hide the cookie banner on the identity pages similarly to the privacy pages: https://github.com/AutoScout24/showcar-tracking/blob/master/src/cmp.js#L138

[the remark this answers is not in the document]
Comment (bdadam) [reply src/cmp.js:169]: (Please note that in the link above there is a bug which I'll fix and add the identity pages)

**Review by bdadam — CHANGES_REQUESTED** [review]

File: src/gtm/getGtmContainerId.js [hunk 3]
```diff
@@ -10,7 +10,18 @@ var containerIdsByTld = {
     com: 'GTM-KWX9NX',
 };
 
+var isIdentityPage = function (hostname) {
+    return hostname === "accounts.autoscout24.com";
+};
+
+var extractTldFromRedirectUrl = function (url) {
+    var regexp = new RegExp(/24\.([a-z]{2,3})/g);
```
[the remark this answers is not in the document]
Comment (mbullin) [reply src/gtm/getGtmContainerId.js:18]: i can try :)

File: src/cmp.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mbullin) [reply src/cmp.js:169]: ah ok good to know will remove it.

**Pull request merged**